src/main.rs

```diff
@@ -144,6 +164,20 @@ fn main() {
         }
         None => None,
     };
+    let mut width: usize = 80;
+    if matches.is_present("width") {
+        let val = matches.value_of("width");
+        let x: Option<&str> = Some("full");
```

Comment:
I don't have enough understanding of Rust's string comparison to compare `val` to a string, so I made a throwaway Option to compare to.